Volume/Supporting Views/SearchTabBar.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  SearchTabBar.swift
+//  Volume
+//
+//  Created by Vian Nguyen on 11/27/22.
+//  Copyright © 2022 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct SearchTabBar: View {
```

Comment:
I didn't realize you were working on something like this too haha. In #118 I created a `SlidingTabBarView` with that you can reuse here to prevent redundant code.